packages/cli/src/cmds/validator/handler.ts

```diff
@@ -169,10 +171,15 @@ export async function validatorHandler(args: IValidatorCliArgs & GlobalArgs): Pr
       distributed: args.distributed,
     },
     metrics
-  );
+  ).catch((err) => {
+    if (isErrorAborted(err)) {
+      throw new Error("Validator initialization aborted. Exiting.");
```

Comment:
Why do we want to throw an error? This will still print out a stack trace and there is not much difference compared to just throwing the abort error

After all, as you mentioned, we are printing "Stopping gracefully.." already

Reply:
It shows a better message to suer why stopping, than simply error stating `aborted`

Reply:
I really don't know about this change, I am not sure we are fixing a real problem here. I never saw a user complaint about this and it shouldn't cause confusion and bad UX if we throw an aborted error here.

As this does not address points mentioned in the issue I would be in favor of removing it from this PR and opening a separate PR.

Maybe can get further opinions on this from others

Reply:
Removed.